internal/replacements/replacements_test.go

```diff
@@ -86,6 +91,28 @@ var _ = Describe("replace_misconfigured_pods", func() {
 
 			pod.Spec = *spec
 
+			//node = &corev1.Node{Name: pod.Spec.NodeName}
+			node = &corev1.Node{}
```

Comment:
Do we plan to make any tests with this setup?

Reply:
nope. removing it